Dockerfile

```diff
@@ -1,30 +1,56 @@
-FROM centos:7
+# Base image inherited by all cfgov-refresh child images
+FROM centos:7 AS cfgov-base
 
-# Install required packages
-RUN yum install -y https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-centos10-10-2.noarch.rpm && \
-    yum install -y epel-release make gcc gcc-c++ kernel-devel mailcap which && \
-    yum install -y postgresql10 postgresql10-devel && \
-    yum install -y python-devel python36 python36-devel && \
-    yum clean all
+SHELL ["/bin/bash", "--login", "-c"]
+
+# Stops Python default buffering to stdout, improving logging to the console.
+ENV PYTHONUNBUFFERED 1
+
+ENV APP_HOME /src/cfgov-refresh
+RUN mkdir -p ${APP_HOME}
+WORKDIR ${APP_HOME}
+
+# Install common OS packages
+RUN yum -y install \
+        centos-release-scl \
+        epel-release \
+        https://download.postgresql.org/pub/repos/yum/10/redhat/rhel-7-x86_64/pgdg-centos10-10-2.noarch.rpm && \
+    yum -y install \
+        gcc \
+        httpd \
```

Comment:
Is this forward-looking for a time when this container can run Apache as well, or is it needed if we're only still using runserver?

Reply:
You're right, that can probably be taken out.  Confirming now...

Reply:
Removed.